.github/workflows/build.yaml

```diff
@@ -0,0 +1,42 @@
+name: build
+on:
+  pull_request:
+env:
+  go-version: 1.23
+jobs:
+  build:
+    name: Build images
+    strategy:
+      matrix:
+        ubuntu-version: ["20.04", "22.04", "24.04"]
+    runs-on: ubuntu-24.04
+    defaults:
+      run:
+        working-directory: ${{ matrix.ubuntu-version }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup golang
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.go-version }}
+      - name: Build images
+        run: |
+          TAG_MINIMAL=$(cat TAG_MINIMAL)
+          TAG=$(cat TAG)
+          
+          echo
+          echo "building ubuntu-minimal:$TAG_MINIMAL..."
+          docker build \
```

Comment:
[Q]
It's only for test so it needn't to run `buildx`, right?

Reply:
Actually, when using buildx, I don't know how to use the image built in the runner to build another image.
If you know, please

Reply:
Sorry, me too. I think it's ok because it's just a build test and `release.yaml` runs `buildx`.